packages/canary/src/components/meter.tsx

```diff
@@ -16,30 +16,22 @@ interface MeterRootProps {
   className?: string
 }
 
-const stateToBgColor = {
+const stateToBgColor: { [key in MeterState]: string } = {
   [MeterState.Empty]: 'bg-tertiary-background/20',
   [MeterState.Error]: 'bg-error',
   [MeterState.Warning]: 'bg-warning',
   [MeterState.Success]: 'bg-success'
 }
 
-const stateToGlow = {
-  [MeterState.Empty]: '', // No glow for empty state
-  [MeterState.Error]: 'shadow-[0_0_6px_2px_hsl(var(--error-glow))]',
-  [MeterState.Warning]: 'shadow-[0_0_6px_2px_hsl(var(--warning-glow))]',
-  [MeterState.Success]: 'shadow-[0_0_6px_2px_hsl(var(--success-glow))]'
-}
+function Root({ data = [], className }: MeterRootProps) {
+  const emptyBarsCount = 11 - data.length
+  const bars = [...Array(emptyBarsCount).fill({ state: MeterState.Empty }), ...data]
 
-function Root({ data, className }: MeterRootProps) {
   return (
     <div className={cn('flex h-[19px] gap-[4px] items-stretch', className)}>
-      {data?.map((col, col_idx) => {
-        return (
-          <div
-            key={col_idx}
-            className={cn('flex w-[5px] h-full rounded-[1px]', stateToBgColor[col.state], stateToGlow[col.state])}
-          />
-        )
+      {bars.map((col, col_idx) => {
```

Comment:
index is not a good key: https://dev.to/shiv1998/why-not-to-use-index-as-key-in-react-lists-practical-example-3e66